azurerm/internal/services/streamanalytics/resource_arm_stream_analytics_job.go

```diff
@@ -162,7 +162,7 @@ func resourceArmStreamAnalyticsJobCreateUpdate(d *schema.ResourceData, meta inte
 
 	// needs to be defined inline for a Create but via a separate API for Update
 	transformation := streamanalytics.Transformation{
-		Name: utils.String("Transformation"),
+		Name: utils.String("main"),
```

Comment:
Would it make sense to instead expose this property with a computed default of "main"? 🤔 that way if the portal changes at some point or the cli is differet code updates won't be required

Reply:
We could do that but i changed the implementation so that when the resource is read the transformation is read also (inline) so that we always have the correct name.

Reply:
this is an implementation detail more than anything, so I don't think it's worth exposing this here?